browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -4475,41 +4480,43 @@ TEST_F(KeyringServiceUnitTest, BitcoinReceiveChangeAddresses) {
       "brave", false));
 
   EXPECT_THAT(
-      service.GetKeyringInfoSync(mojom::kBitcoinKeyringId)->account_infos,
+      service.GetKeyringInfoSync(mojom::kBitcoinKeyring84Id)->account_infos,
       testing::IsEmpty());
 
-  EXPECT_EQ(service.GetBitcoinReceivingAddress(mojom::kBitcoinKeyringId, 0, 0),
-            absl::nullopt);
-  EXPECT_EQ(service.GetBitcoinChangeAddress(mojom::kBitcoinKeyringId, 0, 0),
-            absl::nullopt);
-  EXPECT_EQ(service.GetBitcoinReceivingAddress(mojom::kBitcoinKeyringId, 1, 0),
-            absl::nullopt);
-  EXPECT_EQ(service.GetBitcoinChangeAddress(mojom::kBitcoinKeyringId, 1, 0),
-            absl::nullopt);
+  EXPECT_EQ(service.GetBitcoinAddresses(mojom::kBitcoinKeyring84Id, 0).size(),
+            0u);
 
-  AddAccount(&service, "Btc Acc", mojom::CoinType::BTC);
+  AddBitcoinAccount(&service, "Btc Acc", mojom::kBitcoinMainnet,
+                    mojom::kBitcoinKeyring84Id);
 
   EXPECT_THAT(
-      service.GetKeyringInfoSync(mojom::kBitcoinKeyringId)->account_infos,
+      service.GetKeyringInfoSync(mojom::kBitcoinKeyring84Id)->account_infos,
       testing::SizeIs(1u));
-  auto btc_acc = service.GetKeyringInfoSync(mojom::kBitcoinKeyringId)
+  auto btc_acc = service.GetKeyringInfoSync(mojom::kBitcoinKeyring84Id)
                      ->account_infos[0]
                      ->Clone();
   EXPECT_EQ(btc_acc->address, "bc1ql5f64jdzjsvgehlpxvdgm9ygp0xta7xpnueh03");
   EXPECT_EQ(btc_acc->name, "Btc Acc");
   EXPECT_FALSE(btc_acc->is_imported);
   EXPECT_FALSE(btc_acc->hardware);
   EXPECT_EQ(btc_acc->coin, mojom::CoinType::BTC);
-  EXPECT_EQ(btc_acc->keyring_id, mojom::kBitcoinKeyringId);
+  EXPECT_EQ(btc_acc->keyring_id, mojom::kBitcoinKeyring84Id);
 
-  EXPECT_EQ(service.GetBitcoinReceivingAddress(mojom::kBitcoinKeyringId, 0, 0),
+  EXPECT_EQ(service.GetBitcoinAddresses(mojom::kBitcoinKeyring84Id, 0)[0].first,
             "bc1qcr8te4kr609gcawutmrza0j4xv80jy8z306fyu");
-  EXPECT_EQ(service.GetBitcoinChangeAddress(mojom::kBitcoinKeyringId, 0, 0),
-            "bc1q8c6fshw2dlwun7ekn9qwf37cu2rn755upcp6el");
-  EXPECT_EQ(service.GetBitcoinReceivingAddress(mojom::kBitcoinKeyringId, 1, 0),
-            absl::nullopt);
-  EXPECT_EQ(service.GetBitcoinChangeAddress(mojom::kBitcoinKeyringId, 1, 0),
-            absl::nullopt);
+  EXPECT_EQ(
+      service.GetBitcoinAddresses(mojom::kBitcoinKeyring84Id, 0)[0].second,
```

Comment:
Please check that total is 50 accounts and 0-29 should be all `mojom::BitcoinKeyId::New(0, 0, 0)` and 30-49 should all be `mojom::BitcoinKeyId::New(0, 1, 0)`

Reply:
I'll make proper test when GetBitcoinAddresses gets 'last receive and change address' prefs implemented. Just want to avoid rewriting test.